shared/src/main/scala/mlscript/syntax.scala

```diff
@@ -181,12 +183,15 @@ final case class NuTypeDef(
   nme: TypeName,
   tparams: Ls[(Opt[VarianceInfo], TypeName)],
   params: Tup, // the specialized parameters for that type
+  ctor: Opt[Constructor],
+  isPlain: Bool,
```

Comment:
```suggestion
  isPlainJSClass: Bool,
```

Please use this more understandable name everywhere.